demos/segmentation_using_paddlepaddle_model/python/README.md

```diff
@@ -20,6 +18,13 @@ git clone https://github.com/openvinotoolkit/model_server.git
 cd model_server/demos/segmentation_using_paddlepaddle_model/python
 ```
 
+Install [paddlepaddle](https://pypi.org/project/paddlepaddle/) and other packages required to export the model to deployable format
+```bash
+pip3 install paddlepaddle
+pip3 install pyyaml
+pip3 install scipy
```

Comment:
Can we use venv mechanism like in many other examples?

![image](https://user-images.githubusercontent.com/74778212/194516139-60a5b05d-56ac-4bf8-a7de-83d7ea72fbfd.png)

Reply:
![image](https://user-images.githubusercontent.com/74778212/194516456-a532f246-bd6a-4ad0-8008-693c7df19994.png)

Reply:
we are not enforcing the use of virtual env in other samples, that would be out of scope for this PR

Reply:
can we enclose it in requirements file and add versions to it? this way we could be sure that there would be no problems     with wrong versions in the future